src/Compilers/CSharp/Portable/Binder/UsingStatementBinder.cs

```diff
@@ -90,6 +90,7 @@ internal static BoundStatement BindUsingStatementOrDeclarationFromParts(SyntaxNo
             AwaitableInfo awaitOpt = null;
```

Comment:
>awaitOpt = null [](start = 26, length = 15)

`awaitOpt` could be renamed since it is no longer optional, and the initializer can be removed.